ocaml/testsuite/tests/typing-tailcalls/locals.ml

```diff
@@ -1,37 +1,42 @@
 (* TEST
-   expect;
+  flags = "-less-tco";
+  expect;
 *)
 
 let [@inline never] f (local_ str) = let _ = str in ()
 
-[%%expect {|
-val f : local_ 'a -> unit = <fun>
-|}]
-
 module M = struct
   let f = f
 end
+
 [%%expect {|
+val f : local_ 'a -> unit = <fun>
 module M : sig val f : local_ 'a -> unit end
 |}]
 
+
 let implicit_nontail () =
```

Comment:
Can we also show implicit nontail of names bound by a surrounding `pexp_let`? Something with a shape like
```ocaml
let x = let g x = x in g 5;;
```